docs/guides/building/README.md

```diff
@@ -8,6 +8,6 @@ The Tauri Bundler is a Rust harness to compile your binary, package assets, and
 
 It will detect your operating system and build a bundle accordingly. It currently supports:
 
-- Linux: .deb, .appimage
-- macOS: .app, .dmg
-- Windows: .exe, .msi
+- [Windows](./windows.md): .exe, .msi
```

Comment:
.exe files are not really distributable (i actually thought we dropped that already), so let's not give people the wrong idea.
```suggestion
- [Windows](./windows.md): .msi
```